packages/components/button/src/Button/README.mdx

```diff
@@ -14,6 +14,10 @@ Buttons communicate the action that will occur when the user clicks them. They c
 
 ```js static=true
 import { Button } from '@contentful/f36-components';
+
+// or
+
+import { Button } from '@contentful/f36-button';
```

Comment:
```suggestion
// or
import { Button } from '@contentful/f36-button';
```

I suggest not adding the new line breaks, since you told me the team didn't want the `import`section to be too "tall"